Proposals/xGov-182.md

```diff
@@ -0,0 +1,62 @@
+---
+id: 182
+period: 4
+title: Airgap vault & wallet integration via isolated modules
+author: AP (@Vidhyanandcs)
+email: vidhyanandcs@gmail.com
+discussions-to: https://forum.algorand.org/t/xgov-108-airgap-vault-wallet-integration-via-isolated-modules/11004
+company_name: Securecerts Technologies
+category: Tools
+focus_area: Deployment
+open_source: Yes
+funding_type: Proactive
+amount_requested: 84530
+delivery_date: 2024-08-01
+status: Draft
+---
+
+## Abstract
+
+Airgap vault is an open-source vault that helps algorand community  to convert their old mobile phone to a cold storage for generation and management of private keys. Airgap wallet is an open-source wallet with web, android, iOS, mac, windows and linux support. It currently supports BTC, ETH, XTZ, DOT, GLMR, ATOM, ICP, RBTC & more. Unlike other wallet ecosystems, your private key (seed phrase) is never stored in the wallet app but in the vault. We are proposing a plan to integrate airgap vault and wallet via isolated modules to algorand.
+
+## Team
+
+AP - CEO. I have 7 years of experience in sales and marketing. I have been an active member in the Algorand community since August 2021 and is involved with the Notiboy project.
+
+Deepak -CTO. Deepak has a Masters in computer science and has worked in product design & development for leading payment companies in the world. He is well versed in the Algorand smart contracts and SDKs and has been leading the design and development of Notiboy project.
+
+## Experience with Algorand
+
+We have been building notiboy project which is a notification and web3 chat application on algorand.
+
+## Present Proposal
+
+As My Algo Wallet announced sunset, we were looking for an alternative multi-chain wallet that can be used by community to operate hardware wallet. We stumbled upon airgap which was recommended to us by a community member. It has two features. Airgap vault and Airgap wallet.
+
+The AirGap Vault is responsible for secure secret recovery phrase generation and storage, as well as air-gapped transaction signing through QR codes. The Vault is installed on a dedicated device with no network connection, thereby protecting your secret recovery phrase from attackers. Even if you choose to install the Vault on a device with network connectivity, the Vault application still stays offline.
+
+The AirGap Wallet creates transactions, broadcasts transactions and shows a portfolio overview. The Wallet is installed on your everyday smartphone with network access. The wallet stores public information. The secret recovery phrase is only stored and used in the AirGap Vault, which does not have access to the internet at any time, thus making it inherently more secure than your regular software wallet with internet access.
+
+We intend to integrate airgap vault and airgap wallet to algorand via isolated modules.
+
```

Comment:
Please provide public URL of where/how the community can check proposal deliverables.

Reply:
> Please provide public URL of where/how the community can check proposal deliverables.

Done. Once we do the integration, airgap team will audit the code and update it in the muilti coin lib github repo.
If there is any change in this we shall notify you at that point